tests/validation/tests/v3_api/test_import_cluster.py

```diff
@@ -26,8 +26,7 @@ def test_import_rke_cluster():
     # Create AWS nodes for the cluster
     aws_nodes = \
         AmazonWebServices().create_multiple_nodes(
-            AWS_NODE_COUNT, random_test_name(HOST_NAME),
-            wait_for_ready=True)
+            AWS_NODE_COUNT, random_test_name(HOST_NAME))
```

Comment:
wait_for_ready=True is removed in this file as it is set to true by default ?

Reply:
yes, it is true by default.